<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>635</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>简易数据库代码生成器V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1161</width>
      <height>140</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="txtDBUrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1141</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenDB">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>90</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>打开数据库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMakeCode">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>90</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>生成代码</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxDBAdapters">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>331</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMakeAllCode">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>生成所有代码到文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHelp">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>100</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>关于</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeView" name="tvTables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>341</width>
      <height>470</height>
     </rect>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="twPages">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>801</width>
      <height>470</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabNormal">
     <property name="currentTabText" stdset="0">
      <string>常用代码</string>
     </property>
     <attribute name="title">
      <string>常用代码</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtNormalText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>421</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEntityAndDAO">
     <property name="currentTabText" stdset="0">
      <string>fff</string>
     </property>
     <attribute name="title">
      <string>实体和DAO</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtEntityAndDAOText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>421</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabNormalScript">
     <attribute name="title">
      <string>编辑常用代码脚本</string>
     </attribute>
     <widget class="QTextEdit" name="txtNormalScript">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveNormalScript">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEntityAndDAOScript">
     <attribute name="title">
      <string>编辑实体和DAO代码脚本</string>
     </attribute>
     <widget class="QPushButton" name="btnSaveEntityAndDAOScript">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtEntityAndDAOScript">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="btnConfigEditor">
     <attribute name="title">
      <string>编辑配置文件</string>
     </attribute>
     <widget class="QPushButton" name="btnSaveConfig">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtConfig">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownloadOutputTemplete">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>下载输出模板</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownloadInputTemplete">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>下载输入模板</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
